src/main/kotlin/day17/data_source/pixabayPhoto/PixabayPhotoDataSourceImp.kt

```
@@ -0,0 +1,64 @@
+                    else RResult.Error(PhotoError.ServerError.BadRequest("400 error"))
+                }
+
+                HttpStatusCode.InternalServerError -> {
+                    RResult.Error(PhotoError.ServerError.InternalServerError("Internal Server Error"))
+                }
+
+                else -> {
+                    RResult.Error(PhotoError.ServerError.UnknownError("예상치 못한 에러"))
+                }
+            }
+        } catch (e: IOException) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)